tests/public-api/requirements/_test.txt

```diff
@@ -0,0 +1,111 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile --output-file=requirements/_test.txt requirements/_test.in
+#
+attrs==20.3.0
+    # via
+    #   jsonschema
+    #   pytest
+certifi==2020.12.5
+    # via
+    #   httpx
+    #   osparc
+    #   requests
+chardet==3.0.4
+    # via
+    #   httpx
+    #   requests
+contextvars==2.4
+    # via sniffio
+coverage==5.4
```

Comment:
so you use the latest pip-tools then? shall I change also in my PR?

Reply:
I do not have a strong opinion on that. 
I like the new style (it is more readable) but at the same time I understand that it has limitations with dependabot ... although they seem to have resolve them? 

The important thing is to have them all in the same format. So that said ... perhaps now i have a proposal: we freeze in the old pip-tools and upgrade them all at the same time with the PR of py3.8 where actually all the requirements are affected. What do you think?

Reply:
@sanderegg your call! :-)

Reply:
well since I won't put orjson now.. I will revert the pinning. I guess we need to upgrade anyhow. I do not really care about the formatting. just that it should be done everywhere the same if possible.